src/BrowscapUpdater.php

```diff
@@ -4,27 +4,28 @@
 namespace BrowscapPHP;
 
 use BrowscapPHP\Cache\BrowscapCache;
-use BrowscapPHP\Cache\BrowscapCacheInterface;
 use BrowscapPHP\Exception\FetcherException;
 use BrowscapPHP\Helper\Converter;
+use BrowscapPHP\Helper\ConverterInterface;
 use BrowscapPHP\Helper\Filesystem;
 use BrowscapPHP\Helper\IniLoader;
+use BrowscapPHP\Helper\IniLoaderInterface;
 use GuzzleHttp\Client;
 use GuzzleHttp\ClientInterface;
 use Psr\Log\LoggerInterface;
-use Psr\Log\NullLogger;
-use WurflCache\Adapter\AdapterInterface;
-use WurflCache\Adapter\File;
+use Psr\SimpleCache\CacheInterface;
 
 /**
  * Browscap.ini parsing class with caching and update capabilities
  */
-final class BrowscapUpdater
+final class BrowscapUpdater implements BrowscapUpdaterInterface
 {
+    const TIMEOUT = 5;
```

Comment:
Can this be a `private` const? If not, make it explicitly `public` please to denote usage.

Reply:
In fact, I'd probably name this `DEFAULT_TIMEOUT` too

Reply:
done